code/datums/emergency_calls/inspection.dm

```diff
@@ -9,6 +9,15 @@
 	..()
 	objectives = "Investigate any issues with ML enforcement on the [MAIN_SHIP_NAME]."
 
+/datum/emergency_call/inspection_provost/remove_nonqualifiers(list/datum/mind/candidates_list)
+	var/list/datum/mind/candidates_clean = list()
+	for(var/datum/mind/single_candidate in candidates_list)
+		if(check_timelock(single_candidate?.current?.client, JOB_POLICE, time_required_for_job))
+			candidates_clean.Add(single_candidate)
+			continue
+		if(single_candidate.current)
+			to_chat(single_candidate.current, SPAN_WARNING("You didn't qualify for the ERT beacon because you dont have enough playtime (5 Hours) as military police!"))
```

Comment:
```suggestion
			to_chat(single_candidate.current, SPAN_WARNING("You didn't qualify for the ERT beacon because you don't have enough playtime (5 Hours) as military police!"))
```